### PR TITLE
Updated js and functionality to match that of the accordion element.

### PR DESCRIPTION
**Pull request by M-Porter**
**Changes**
* When a user adds elements to the content areas, the content area reopens and resizes on page load
* Content areas resize as text is added.
* Smoother overall experience

### CODE REVIEW

File: files/js/editor_element.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+ * This is required for element rendering to be possible
+ * @type {PlatformElement}
+ */
+(function() {
+    var FAQ = PlatformElement.extend({
+        initialize: function() {
+            this.activeIndex = this.settings.get('active_index');
+
+            this.fixStyles();
+            this.setupAccordion();
```
Comment (puddi) [inline files/js/editor_element.js]: might want to rename this.

File: files/js/editor_element.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+            });
+        },
+
+        /**
+         * If an index was active when the page was
+         * reloaded or refreshed, open it back up
+         */
+        setOpen: function() {
+            /**
+             * "NA" is the identifier that no items
```
Comment (puddi) [inline files/js/editor_element.js]: not a big deal (since it's in a comment), but the change from NA to -1 didn't get reflected here.

**Comment by puddi** [comment]
:+1:

File: files/js/editor_element.js (hunk 1 above)
Comment (M-Porter) [inline files/js/editor_element.js]: I'm keeping it because it technically still is an accordion element.

**Pull request merged**